crates/astria-composer/src/executor/mod.rs

```diff
@@ -521,6 +557,7 @@ pin_project! {
 impl Future for SubmitFut {
     type Output = eyre::Result<u32>;
 
+    #[allow(clippy::too_many_lines)]
```

Comment:
Can you make a tracking issue for us to refactor this? So we don't lose track, i.e.:
```suggestion
    // allow: refactor this future github.com/astriaorg/astria/issues/{number}
    #[allow(clippy::too_many_lines)]
```